framework/src/modules/network/components/storage/sql/migrations/runtime.sql

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+
+/*
+  DESCRIPTION: Runtime queries - executed at node start (right after migrations).
+
+  PARAMETERS: None
+*/
+
+-- Overwrite unconfirmed tables with state from confirmed tables
+DROP TABLE mem_accounts2u_delegates;
```

Comment:
probably we don't need this

Reply:
removed migration stuff completely from network module